mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -15,5 +15,6 @@
 <!ENTITY aboutSupport.accountsConnSecurity            "Bezpieczeństwo połączenia">
 <!ENTITY aboutSupport.accountsAuthMethod              "Metoda uwierzytelniania">
 <!ENTITY aboutSupport.accountsDefault                 "Domyślne?">
+<!ENTITY aboutSupport.identityName                    "Tożsamość">
```

Comment:
I tried to always keep the surrounding formatting. I can unify the spaces across the files, if you want.